go/test/endtoend/reparent/utils/utils.go

```diff
@@ -129,7 +129,7 @@ func setupCluster(ctx context.Context, t *testing.T, shardName string, cells []s
 		// the replication manager to silently fix the replication in case ERS or PRS mess up. All the
 		// tests in this test suite should work irrespective of this flag. Each run of ERS, PRS should be
 		// setting up the replication correctly.
-		"--disable_active_reparents")
+		"--disable-replication-manager")
```

Comment:
This change is required because in the tests we test that a tablet that is restarted starts replicating from the primary again. So we want the replication initialization code to run. `disable_active_reparents` blocks both the replication manager and the replication initialization, but `disable-replication-manager` only blocks the first.

We required this change after increasing the onclose timeout. Earlier, because onclose timeout was 1ns, the vttablets when shutting down wouldn't have the time to remove the hostname, port from the vttablet records. This caused a call to ERS to have a hanging SetReplicationSource. This call would succeed when the tablet was up again. But this is incorrect and not want the test really wanted to test.

After increasing the onclose timeout, the vttablets now remove their hostname, port information from the tablet records. This causes the SetReplicationSource call to be directed at a tablet with address `:0`. So that call doesn't setup the replication. Instead, we require the replication initialization to do it properly. This is the reason for this change.

These changes resolve flakiness in the ERS tests which showed itself as `vttablet timed out after 10s` failing to reach the Serving state, in the few cases that onclose function did complete before timing out.